cli.js

```diff
@@ -65,6 +67,13 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
         describe: 'Function name',
         type: 'string',
       })
+      .option('executionModel', {
+        alias: 'e',
+        choices: executionModels,
+        describe: 'Execution model for your function',
+        default: 'exclusive',
```

Comment:
the default should not be determined here. if the flag is missing just don't alter the templated yml. in the future this will be automatically detected unless specified, so no specification is a strong and important signal from the user (iiuc, using undefined as a default should be an easy solution)